components/amorphic/lib/utils/AmorphicUtils.ts

```diff
@@ -0,0 +1,37 @@
+export class AmorphicUtils {
+
+    static toBool(value, defaultValue: boolean = false): boolean {
+        switch (typeof value) {
+          case 'undefined':
+            return defaultValue;
+          case 'boolean':
+            return value;
+          case 'string':
```

Comment:
Just a quick comment, an empty string in this case will evaluate to True, while Boolean('') is actually false. Is that alright here?

Reply:
Good point. I had a rethink on this one after your comment. Resolving this to only evaluate true if string value of 'true' or boolean value of true is set and false in all other case.